packages/manager/modules/product-offers/src/workflows/product-offers-services-workflow.class.js

```diff
@@ -40,8 +41,10 @@ export default class ServicesWorkflow extends Workflow {
     this.pricings = this.computePricing(detachPlancode.prices);
 
     if (this.hasUniquePricing()) {
-      this.currentIndex += 1;
-      [this.pricing] = this.pricings;
+      this.$timeout(() => {
```

Comment:
I don't understand the use of `$timeout` here

Reply:
@Jisay the `$timeout` is necessary to increment correctly the currentIndex (exposed variable of `oui-stepper`), after the component is initialized

Reply:
@Jisay @JeremyDec I think it's ok as a workaround but we should try to find why this `$timeout` is needed and if we can avoid its usage